### ISSUE TITLE
Dependencies changed in 0.12.0

### ISSUE THREAD

**Issue by vikahl** [Kludex/uvicorn#788]
### Checklist

- [X] The bug is reproducible against the latest release and/or `master`.
- [X] There are no similar issues or pull requests to fix it yet.

### Describe the bug

Version 0.12.0 removes some dependencies that causes our services to crash.
The removed dependencies are not listed in the changelog.

### To reproduce

```shell
$ pip3 install uvicorn==0.11.8 && pip3 freeze
click==7.1.2
h11==0.9.0
httptools==0.1.1
uvicorn==0.11.8
uvloop==0.14.0
websockets==8.1
```

```shell
$ pip3 install uvicorn==0.12.0 && pip3 freeze
click==7.1.2
h11==0.10.0
uvicorn==0.12.0
```

### Expected behavior

`uvloop` and `httptools` to be installed.

### Actual behavior

`uvloop` and `httptools` was not installed.

### Environment

Tested on Python 3.8.3, pip 19.2.3.

**Comment by vikahl** [comment]
I found https://github.com/encode/uvicorn/pull/666 which seems to be the source, so it solves my issue.
It would be great if these changes was included in the changelog though.

**Comment by florimondmanca** [comment]
Hi @vikahl!

Thanks for reporting this :)

I think given the tight constraints on developer resources for Uvicorn for now, and if you’ve got any chance, a PR making the changelogs and/or docs updates that would have helped you figure out what to do to migrate to 0.12.0 (ie using the new optional deps) would be _greatly_ appreciated. Thanks!

**Comment by vikahl** [comment]
> I think given the tight constraints on developer resources for Uvicorn for now, and if you’ve got any chance, a PR making the changelogs and/or docs updates that would have helped you figure out what to do to migrate to 0.12.0 (ie using the new optional deps) would be _greatly_ appreciated. Thanks!

I think the readme in the repository explains how to install the package (using `[standard]` extras), but I usually only read the changelog to see if it's safe to upgrade.

I issued a PR adding a comment about the change in packaging, which would have been enough for me.

(and btw, great job with the package)

**Comment by euri10** [comment]
fixed in #792
